recipes/spyder-vim/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "spyder-vim" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-vim-{{ version }}.tar.gz
+  sha256: 97c2ef31ae9bb24af1853d3741081cc27867f4c170e031a07daa213cc79577ca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - qtpy
+    - qtawesome
```

Comment:
Some reasonable lower bound here might be helpful...

Reply:
Lets keep it synched with upstream

Reply:
Yeah, only problem is if we do change it later, conda could backsolve to these old uncapped versions (which may or may not actually work).